messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/AddressSettingAdd.java

```diff
@@ -94,12 +95,11 @@ static AddressSettings createDefaulAddressSettings() {
      * Create a setting.
      *
      * @param context the operation context
-     * @param config the detyped config
+     * @param config  the detyped config
      * @return the address settings
-     *
      * @throws OperationFailedException if the model is invalid
      */
-    static AddressSettings createSettings(final OperationContext context, final ModelNode config) throws OperationFailedException {
+    static AddressSettings createSettings(final OperationContext context, final ModelNode config, boolean isRootAddressMatch) throws OperationFailedException {
```

Comment:
I don't really see the value of this boolean because the default values will be set by the merge